src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -1,20 +1,54 @@
 import React from 'react';
 import { Field } from 'redux-form';
-import PropTypes from 'prop-types';
+import PropTypes, { arrayOf } from 'prop-types';
 import RenderInputTextField from '../RenderInputTextField';
 import FieldLabel from '../FieldLabel';
 import RichEditor from '../RichEditor';
 import { utcTimeZone } from '../../utils';
 import DateTimeField from '../DateTimeField';
+import RenderSelectField from '../RenderSelectField';
+import ReduxFormCreatableSelect from '../ReduxFormCreatableSelect';
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
 
 const AdditionalMetadataFields = (props) => {
   const {
-    disabled, sourceInfo, externalCourseMarketingType,
+    disabled, sourceInfo, externalCourseMarketingType, additionalMetadataOptions,
   } = props;
   const sourceSlug = sourceInfo?.slug;
 
   const SOURCE_SLUG_REQUIRED_FIELDS = JSON.parse(process.env.ADDITIONAL_METADATA_REQUIRED_FIELDS);
 
+  const courseTagObjectsToSelectOptions = (allCourseTags) => allCourseTags.map(tag => ({
+    label: tag.value,
+    value: tag.value,
+  })).filter(x => x.value);
```

Comment:
There might be an existing util for this because there is a tag component for Course already.